listView.php

```diff
@@ -5,37 +5,90 @@
 <head>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
     <title>Student Planner</title>
+</head>
 
     <body>
         <h1 class="text-center">Student Planner</h1>
 	<br>
         <h2 class="text-center">List View</h2>
 	<br>
 
-    </body>
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#createTask">
+  Create Task
```

Comment:
The button says 'Create Task', but when clicked it instructs to 'Add a new list' instead. If this was actually made to create a new task, then it would be redirected to the task creation page, or else this can be renamed to 'Create List'.